### DISCUSSION TITLE
Property descriptions?

### DISCUSSION THREAD

**Discussion by axefrog** [jgraph/drawio#3421] [General]
Shapes seem to have a lot of properties, some whose purpose is non-obvious, but there doesn't seem to be any documentation anywhere describing what each property does, and (where applicable) how it relates to some of its sibling properties.

![image](https://user-images.githubusercontent.com/298883/224177563-9bfdd834-5b8d-4e74-89b6-abcbff1d0856.png)

Is there actually some documentation hidden away somewhere, and if not, who can I ask? I was surprised to find that there was no feature request in the issues list for some better documentation.

**Comment by joshwand** [comment]
+1, this was also reported in https://github.com/jgraph/drawio-desktop/issues/429
